Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,108 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  Git_URL=""
+  Git_SHA=""
+  Ruby_URL=""
+  Ruby_SHA=""
+else
+  Git_URL=""
+  Git_SHA=""
+  Ruby_URL=""
+  Ruby_SHA=""
+fi
+
+cleanup-old() {
+  for version in "$HOMEBREW_LIBRARY/Vendor/$VENDOR_NAME/Versions"/*
```

Comment:
FWIW, All of current directories in HOMEBREW_LIBRARY has a upper case in first letter.